source/administration/object-management.rst

```diff
@@ -220,6 +220,33 @@ Conversion to or from XML may be required for importing rules created on S3 or s
 
 See :ref:`minio-lifecycle-management` for more complete documentation.
 
+Target Bucket Configuration for Tiering
+---------------------------------------
+
+Keep in mind the following considerations when configuring the target bucket for tiering:
```

Comment:
I would break this up into two H2s or defnlists. Or just drop this intro and go straight into

- The remote target does *not* require matching object management or versioning configurations to the source

- If you set object locking rules on the remote, these can prevent delete operations on the source from completing
- If you set object expiration rules on the remote, these may result in data loss
- If you enable versioning on the remote, this may result in excess data remaining after delete operations on the source

Something like that.

Reply:
Mostly so that it's clearer what the cause-effect is to any given config.

Reply:
Incorporated this and the comment above on results of target versioning into a reworked section.